readme.md

```diff
@@ -135,7 +135,7 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer:** 'path' pulls in the path of the document and it is an node package that comes with node.
```

Comment:
`path` has _methods_ that will pull in the name of the current file. You are right to say that it is a package that comes with Node.